src/main/java/com/minecolonies/entity/ai/citizen/lumberjack/Tree.java

```diff
@@ -168,8 +168,8 @@ public static boolean checkTree(@NotNull IBlockAccess world, BlockPos pos)
      */
     @NotNull
     private static Tuple<BlockPos, BlockPos> getBottomAndTopLog(
-                                                                 @NotNull IBlockAccess world, @NotNull BlockPos log, @NotNull LinkedList<BlockPos> woodenBlocks, BlockPos bottomLog,
-                                                                 BlockPos topLog)
+                                                                 @NotNull final IBlockAccess world, @NotNull final BlockPos log, @NotNull final LinkedList<BlockPos> woodenBlocks, final BlockPos bottomLog,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Split this 204 characters long line (which is greater than 180 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00103)